README.md

```diff
@@ -42,6 +44,7 @@ If you'd like to have a similar SPA wrapper for the OSS version of FingerprintJS
 
 - [Documentation](#documentation)
```

Comment:
Please align the order of sections according to our template: https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc 

* TOC
* Requirements
* How to install
* Getting started 
... 
*  Additional resources / Documentation

Reply:
Done!